src/editions/transitioning-an-existing-project-to-a-new-edition.md

```diff
@@ -21,7 +21,7 @@ trait Foo {
 ```
 
 This code uses an anonymous parameter, that `Box<Foo>`. This is [not
-supported in Rust 2018](rust-2018/trait-system/no-anon-params.html), and
+supported in Rust 2018](/rust-2018/trait-system/no-anon-params.html), and
```

Comment:
we cannot use links like this, as this stuff has to work offline in the main distribution

Reply:
How do you suggest I move forward?

Links on the website are broken because they are considered relative links without that leading slash. 

`mdbook serve` makes it available on your local machine even when you are "offline", but it sounds like I'm missing some other requirement that you might be aware of.